libreoffice-core/sw/source/core/doc/DocumentContentOperationsManager.cxx

```diff
@@ -4120,9 +4120,7 @@ bool DocumentContentOperationsManager::DeleteAndJoinWithRedlineImpl(SwPaM & rPam
     {
         auto pRedline(std::make_unique<SwRangeRedline>(RedlineType::Delete, rPam));
         if (pRedline->HasValidRange())
-        {
             redlines.push_back(std::move(pRedline));
-        }
```

Comment:
It seems like your changes for removing the Insert comment got removed in your latest force push.

Also prefer use { } for `if` `else`, makes reading the code a lot more straight forward.

Reply:
I don't believe those changes were necessary with the changes I made to the other file. I'll triple check though

Reply:
Ah, if that's the case, you can update the comment to reflect it's for both. Just wanted to make sure it wasn't on accident.